src/main/java/lotto/domain/LotteryNumbersGenerator.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class LotteryNumbersGenerator {
+  private static final List<LotteryNumber> LOTTERY_NUMBERS = init();
+
+  private static List<LotteryNumber> init() {
+    List<LotteryNumber> lotteryNumbers = new ArrayList<>();
+    IntStream
+        .range(LotteryNumber.getMinNumber(), LotteryNumber.getMaxNumber())
```

Comment:
위에서 언급했지만, 이런 경우 새로운 인스턴스를 받을 경우 명사구를 쓰시는게 좋습니다. 
```suggestion
        .range(LotteryNumber.minNumber(), LotteryNumber.maxNumber())
```
이런 식으로 말이죠. 아니면 불변식이 보장된다는 가정하에 Integer.MAX_VALUE처럼 사용할 수 도있겠죠 ㅎ